packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -66,6 +67,8 @@
   "ssi_information_request_page_form_tax_registration_number_label": "TRN",
   "ssi_information_request_page_form_contact_first_name_label": "Contact first name",
   "ssi_information_request_page_form_contact_last_name_label": "Contact last name",
+  "ssi_information_request_page_form_contact_bsn_label": "Contact BSN",
```

Comment:
is this discussed with Maarten? i am not sure if the bsn is for the contact

Reply:
yes. he has confirmed it